csharp/autobuilder/Semmle.Autobuild.Shared/BuildActions.cs

```diff
@@ -216,12 +217,18 @@ XmlDocument IBuildActions.LoadXml(string filename)
 
         public string EnvironmentExpandEnvironmentVariables(string s) => Environment.ExpandEnvironmentVariables(s);
 
-        public void DownloadFile(string address, string fileName)
+        static async Task DownloadFileAsync(string address, string filename)
         {
-            using var webClient = new WebClient();
-            webClient.DownloadFile(address, fileName);
+            using var httpClient = new HttpClient();
```

Comment:
The `HttpClient` is not supposed to be created and disposed per call. But I guess this is not a high throughput webapp where this would matter.